test/MLOps.NET.Tests/MLOps.NET.Tests.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I've started enforcing warnings as errors so you may want to add this in the test project under property group
The CI build doesn't seem to generate warnings for the test project (which is good), but I did have some when building locally.

```
    <!--Do not enforce missing XML warnings on test project -->
    <NoWarn>1591</NoWarn>
```

Reply:
Added this to the Directory.Build.props in the test folder so it applies to all test projects